TCC.Tests/BaseTest.cs

```diff
@@ -17,68 +20,78 @@ public class CompressTest
         [InlineData(PasswordMode.InlinePassword)]
         [InlineData(PasswordMode.PasswordFile)]
         [InlineData(PasswordMode.PublicKey)]
-        public void CompressDecompress(PasswordMode mode)
+        public async Task CompressDecompress(PasswordMode mode)
         {
+            var e = new ExternalDependecies();
+            await e.EnsureAllDependenciesPresent();
+
             string toCompressFolder = TestHelper.NewFolder();
             string compressedFolder = TestHelper.NewFolder();
             string decompressedFolder = TestHelper.NewFolder();
             string keysFolder = TestHelper.NewFolder();
 
             var data = TestData.CreateFiles(1, 1, toCompressFolder);
-            var compressOption = data.GetTccCompressOption(mode, compressedFolder);
+            var compressOption = data.GetTccCompressOption(compressedFolder);
 
             switch (mode)
             {
                 case PasswordMode.None:
                     break;
                 case PasswordMode.InlinePassword:
-                    compressOption.Password = "1234";
+                    compressOption.PasswordOption = new InlinePasswordOption() {Password = "1234"};
                     break;
                 case PasswordMode.PasswordFile:
                     string passfile = Path.Combine(keysFolder, "password.txt");
                     TestHelper.FillFile(passfile, "123456");
-                    compressOption.PasswordFile = passfile;
+
+                    compressOption.PasswordOption = new PasswordFileOption() {PasswordFile = passfile};
                     break;
                 case PasswordMode.PublicKey:
                     {
-                        var p1 = TestHelper.CreateKeyPairCommand("keypair.pem", KeySize.Key4096).Run(keysFolder, CancellationToken.None);
-                        var p2 = TestHelper.CreatePublicKeyCommand("keypair.pem", "public.pem").Run(keysFolder, CancellationToken.None);
-                        var p3 = TestHelper.CreatePrivateKeyCommand("keypair.pem", "private.pem").Run(keysFolder, CancellationToken.None);
-                        compressOption.PublicPrivateKeyFile = Path.Combine(keysFolder, "public.pem");
+                        var p1 = await TestHelper.CreateKeyPairCommand("keypair.pem", KeySize.Key4096).Run(keysFolder, CancellationToken.None);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'p1' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)